storage/encoding/ssz.go

```diff
@@ -62,45 +61,3 @@ func (SSZValueCodec[T]) Stringify(value T) string {
 func (SSZValueCodec[T]) ValueType() string {
 	return "SSZMarshallable"
 }
-
-// SSZVersionedValueCodec provides methods to encode and decode SSZ values for a specific version.
-type SSZVersionedValueCodec[T constraints.SSZMarshallable] struct {
```

Comment:
no longer used, was only used by the ExecPayloadHeader, which didn't even need to be versioned.